oop/homework.py

```diff
@@ -37,33 +37,63 @@ class Cat:
     * Implement get_average_speed and return average_speed
 
     """
+    saturation_level = 50
```

Comment:
Are you sure that this variable has to be a class variable?
Read about the difference between class and instance variables